.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Please provide a clear and concise description of the Issue:**
+
+**Environment (please complete the following information):**
+ - OS: <!-- E.g. macOS, Ubuntu 19.04, etc -->
```

Comment:
I think we should mention generic command which gives us OS and Nextflow versions 

```
nextflow info
```